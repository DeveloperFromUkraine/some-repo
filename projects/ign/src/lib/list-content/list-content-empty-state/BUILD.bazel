load("@angular//:index.bzl", "ng_module")
load("@io_bazel_rules_sass//:defs.bzl", "sass_binary")

package(default_visibility = ["//visibility:public"])

sass_binary(
    name = "style",
    src = "list-content-empty-state.scss",
)

ng_module(
    name = "list-content-empty-state-module",
    srcs = [
        "list-content-empty-state.module.ts",
        "list-content-empty-state.component.ts",
        "index.ts"
    ],
    assets = [":list-content-empty-state.css", "list-content-empty-state.html"],
    tsconfig = "//:tsconfig.json",
    deps = [
        "@npm//tslib",
        "@npm//@types",
        "@angular//packages/core",
        "@angular//packages/common",
    ]
)
