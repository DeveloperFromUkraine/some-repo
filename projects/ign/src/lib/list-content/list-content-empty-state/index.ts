export * from './list-content-empty-state.module';
