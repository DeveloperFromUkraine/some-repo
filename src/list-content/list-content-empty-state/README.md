## API
    ### Selector
        * ign-list-content-empty-state
    
    ### Exported As 
        * ListContentEmptyStateComponent

## Implementation HTML
    <ign-list-content-empty-state></ign-list-content-empty-state>
    
## Implementation TS
    None