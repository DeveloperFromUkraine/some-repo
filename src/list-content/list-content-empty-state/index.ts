export * from './list-content-empty-state.component';
