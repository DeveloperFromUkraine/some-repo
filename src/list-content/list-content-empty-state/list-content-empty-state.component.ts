import { Component } from '@angular/core';

@Component({
  selector: 'ign-list-content-empty-state',
  templateUrl: './list-content-empty-state.html',
  styleUrls: ['./list-content-empty-state.scss'],
})
export class ListContentEmptyStateComponent {}
