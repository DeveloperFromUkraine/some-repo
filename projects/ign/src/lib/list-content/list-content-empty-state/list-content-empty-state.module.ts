import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';

import { ListContentEmptyStateComponent } from './list-content-empty-state.component';

@NgModule({
    imports: [
        CommonModule
    ],
    exports: [
        ListContentEmptyStateComponent
    ],
    declarations: [
        ListContentEmptyStateComponent
    ]
})
export class ListContentEmptyStateModule {

}
