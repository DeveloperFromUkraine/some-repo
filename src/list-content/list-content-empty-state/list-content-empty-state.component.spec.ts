import { ComponentFixture, TestBed } from '@angular/core/testing';
import { ListContentEmptyStateComponent } from './list-content-empty-state.component';
import { ComponentTest } from '../../../test/test-bed/component';
import { Component } from '@angular/core';

describe('List Content Empty State', () => {
  let fixture: ComponentFixture<ListContentEmptyStateComponent>;

  beforeEach(() => {
    ComponentTest.createTestBed([], [ListContentEmptyStateComponent] as Component[]);

    fixture = TestBed.createComponent(ListContentEmptyStateComponent);
  });

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
