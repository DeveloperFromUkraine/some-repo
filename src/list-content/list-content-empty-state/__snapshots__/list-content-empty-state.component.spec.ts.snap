// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListContentEmptyStateComponent should match snapshot 1`] = `
<ign-list-content-empty-state>
  
</ign-list-content-empty-state>
`;
